src/mbgl/gl/fence.hpp

```diff
@@ -12,7 +12,7 @@ class Fence {
     ~Fence();
 
     void insert() noexcept;
-    bool isSignaled() const noexcept;
+    bool isSignaled() const;
```

Comment:
@louwers the caller of this function is not `noexcept`

Reply:
If I am not mistaken `updateInFlight` calls it which is used in 

```
void defragment(const std::shared_ptr<gl::Fence>& fence) noexcept override
```

Reply:
I think `defragment` is erroneously marked `noexcept`.